architecture/workflows.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Workflows
+summary: Techniques for building stateful business workflows in distributed systems
+reviewed: 2023-07-18
+callsToAction: ['solution-architect']
+---
+
+Business processes typically involve various distributed components that need to be invoked in a coordinated manner. The Particular Service Platform supports both choreography and orchestration of workflows.
+
+Note: Choreography and orchestration are not mutually exclusive. The patterns may be combined at different levels of business workflows.
+
+## Choreography
+
+Choreographed workflows are implemented by an implicit flow of events between services instead of a central owner of a process. Services are highly decoupled by the use of [publish/subscribe](/architecture/messaging.md#communication-styles-publishsubscribe-pattern). Published messages are called _events_ because they describe a fact about something that happened to the rest of the system. Subscribers react to events as required. There is no central state of the workflow.
+
+![](/serviceinsight/images/overview-sequence-diagram.png)
+
+A choreographed event-driven workflow across multiple endpoints (visualized by a [ServiceInsight sequence diagram](/serviceinsight/sequence-diagram))
+
+### Implementing choreographed workflows
+
+NServiceBus provides a simple [publish/subscribe](/nservicebus/messaging/publish-subscribe/publish-handle-event.md) API for all supported messaging technologies. NServiceBus can automatically create and manage the necessary infrastructure like topics, subscriptions, and queues.
+
+![](nsb-pub-sub.png)
+
+[**Sample: Publish/Subscribe with NServiceBus →**](/samples/pubsub/native/)
+
+### Challenges
+
+* When implementing complex workflows, messages flowing through the system can quickly become difficult to understand and track. To have a full picture of the message flow, the implementation of every service needs to be known. The absence of a central processor requires more effort to detect failed or stuck business processes. [Particular Service Platform monitoring](/monitoring/) helps to understand and monitor complex choreographed workflows.
```

Comment:
Optional: also link to SI docs about visualization, or somehow split this bullet into 2, one to deal with monitoring and one to deal with visualization as it applies to the "full picture of the message flow".

Reply:
given SP now offers sequence diagram views as well, it seemed to be less valuable to mention both tools at this point.